src/components/Modal/VotingPowerDelegationDetail/VotingPowerDelegationHandler.tsx

```diff
@@ -14,36 +14,43 @@ interface Props {
   userVP: number
 }
 
-function VotingPowerDelegationButton({ candidateAddress, userVP }: Props) {
+function VotingPowerDelegationHandler({ candidateAddress, userVP }: Props) {
   const t = useFormatMessage()
   const [isLoading, setLoading] = useState(false)
+  const [isError, setIsError] = useState(false)
   const { isContractUsable, delegatedAddress, isGlobalDelegation, setDelegate, clearDelegate } =
     useSnapshotDelegateContract()
 
   const isRevocable = !!delegatedAddress && delegatedAddress.toLowerCase() === candidateAddress.toLowerCase()
   const isDelegatedGlobally = isRevocable && isGlobalDelegation
+  const showDelegatedInfo = isRevocable && !isLoading && !isError
 
   const handleClick = async () => {
     setLoading(true)
-
+    setIsError(false)
     try {
       if (isRevocable) {
         await clearDelegate()
       } else {
         await setDelegate(candidateAddress)
       }
-    } catch (error) {
+    } catch (error: any) {
```

Comment:
Can you type this as `Error`? Or maybe add a condition in the catch
```
if(error instanceOf error) {
  ...
}
```

Reply:
It can't be done, it must be any

![image](https://user-images.githubusercontent.com/45410089/168105618-c123821b-82b9-4241-bc17-c3715e348357.png)

https://stackoverflow.com/questions/69021040/why-catch-clause-variable-type-annotation-must-be-any